demisto_sdk/tests/test_files/valid-beta-integration.yml

```diff
@@ -46,3 +46,5 @@ script:
   type: python
   subtype: python3
 beta: true
+tests:
+  - No tests
```

Comment:
You will probably want to add some test files that are explicitly invalid for your unit tests.